m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2751,10 +2790,10 @@ PROCEDURE ERRORHANDLER
    ERRORDETAIL(2)=ERRORDETAIL(2)+": Pref Access type 3 not found"
   ELSE IF WARNINGFOUND<>FALSE THEN
    ERRORDETAIL(2)=ERRORDETAIL(2)+FORMAT(": Acct Warning ###9",WARNINGFOUND)
-  ELSE IF BNOSHARECOUNT=0 AND BNOLOANCOUNT=0 THEN
-   ERRORDETAIL(2)=ERRORDETAIL(2)+": No eligible transfer from shares/loans"
-  ELSE
+  ELSE IF INVALIDACCTTYPEFOUND=TRUE THEN
    ERRORDETAIL(2)=ERRORDETAIL(2)+FORMAT(": Acct Type ####9",ACCOUNT:TYPE)
```

Comment:
original code type-o: 5 places not needed for the account type (####9)

Reply:
Thanks. This will be in the next commit.